Week-1/cattyboard/cattyboard.htm

```diff
@@ -0,0 +1,187 @@
+<html>
+<style>
+  * {
+    box-sizing: border-box;
+  }
+
+  header {
+    float: left;
+    display: flex;
+    background-color: #3a3b3c;
+    padding-left: 25px;
+    width: 100%;
+    text-align: left;
+
+    color: white;
+  }
+
+  nav {
+    position: absolute;
+    float: left;
+    width: 3em;
+    height:85%;
+    background: #a9a9a9;
+    padding: 10px 10px;
+  }
+  li button {
+
+    border: None;
+    outline: None;
+    background: none;
+
+  }
+  p button {
+    padding-left:2em;
```

Comment:
Why did you use em and why not px or rem ?

Reply:
Associated with P tag font size.